web/src/stream_popover.js

```diff
@@ -81,6 +83,16 @@ export function hide_stream_popover() {
     }
 }
 
+function is_open_stream_info() {
+    return Boolean(stream_info_popover_instance);
+}
+export function hide_strem_info_popover() {
```

Comment:
Whoops spelling error
```suggestion
export function hide_stream_info_popover() {
```